app/src/main/java/com/graeberj/taskman/agenda/data/model/Agenda.kt

```diff
@@ -0,0 +1,5 @@
+package com.graeberj.taskman.agenda.data.model
+
+data class Agenda(
+    val items: List<AgendaItem>
+)
```

Comment:
The wrapper isn't necessary, if the only item is a list of agenda items :) You can return that right away